pallets/cash/src/lib.rs

```diff
@@ -101,8 +99,10 @@ pub trait Config:
         + UnfilteredDispatchable<Origin = Self::Origin>
         + GetDispatchInfo;
 
-    /// Convert implementation for Moment -> Timestamp.
-    type TimeConverter: Convert<<Self as pallet_timestamp::Config>::Moment, Timestamp>;
+    /// Gets the most recent timestamp and converts it from a moment
+    type GetConvertedTimestamp: timestamp::GetConvertedTimestamp<
+        <Self as pallet_timestamp::Config>::Moment,
```

Comment:
kind of meh that cash still has a direct dependency on pallet_timestamp. Is it possible to paper over this part too?

Reply:
as in, can Timestamp: The Crate take over completely so that Cash: The Pallet doesn't know about Timestamp: The Pallet

Reply:
hm maybe

Reply:
don't we also use timestamps for calculating yield?